src/main/java/com/scottlogic/practice/mealscheduler/Services/MealService.java

```diff
@@ -0,0 +1,22 @@
+package com.scottlogic.practice.mealscheduler.Services;
+
+import com.scottlogic.practice.mealscheduler.Models.Meal;
+import com.scottlogic.practice.mealscheduler.Repositories.MealRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
```

Comment:
If we're going to extract the beans to a config do we still need this?

Reply:
Yes, we should go this or that way, not both :)